src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/takecard/events/EditTakeEvent.kt

```diff
@@ -0,0 +1,16 @@
+package org.wycliffeassociates.otter.jvm.app.widgets.takecard.events
+
+import javafx.event.Event
+import javafx.event.EventType
+import org.wycliffeassociates.otter.common.data.workbook.Take
+
+class EditTakeEvent(
+    type: EventType<EditTakeEvent>,
```

Comment:
Do you need to accept type in the constructor?  Can you just use the companion value?

Reply:
Done for all